### ISSUE TITLE
How big is the scope of the translation?

### ISSUE THREAD

**Issue by bendtherules** [hchasestevens/xpyth#1]
This is an ideological question rather than any bug. I like the problem you are trying to solve, but am a little sceptical about it and hence this question.

How big do you feel is the scope of the translation from native python to xpath? For eg., its ok if you have a equality check as they are available in xpath. But what if I use a function for the check condition or. even simply suppose sqrt or other non-basic mathematical operations. Or maybe, I look for a value in a dictionary corresponding to the id of all the spans in the document to match something (something like a.id for a in span if (a.something == dict[a]))?
Basically, anything with a dynamic condition on the RHS. How do you plan to handle them?
